data/brands/amenity/restaurant.json

```
@@ -2177,6 +2177,17 @@
         "name": "Honest Burgers"
       }
     },
+    {
+      "displayName": "Honest Greens",
+      "id": "honestgreens-acf031",
+      "locationSet": {"include": ["es"]},
```

```suggestion
      "locationSet": {"include": ["es", "pt"]},
```

@codeinabox mentioned Portugal.